src/pages/flood-control-projects/map.tsx

```diff
@@ -130,9 +135,9 @@ const FloodControlProjectsMap: React.FC = () => {
 
     try {
       await exportMeilisearchData({
-        host: MEILISEARCH_HOST,
-        port: MEILISEARCH_PORT,
-        apiKey: MEILISEARCH_SEARCH_API_KEY,
+        host: 'http://localhost',
+        port: '7700',
+        apiKey: 'masterKey',
```

Comment:
please use the env variable, also using it this way is a massive security risk

Reply:
done